waiter/src/waiter/settings.clj

```diff
@@ -30,9 +30,6 @@
                                              :kind s/Keyword
                                              s/Keyword schema/require-symbol-factory-fn}
                                             schema/contains-kind-sub-map?)
-   (s/required-key :ejection-config) {(s/required-key :eject-backoff-base-time-ms) schema/positive-int
```

Comment:
Lexicographic order. Content only moved, did not change. Not directly related to this PR.